src/v3/test/integration/__snapshots__/oda-enrollment-android-applink.test.tsx.snap

```diff
@@ -347,8 +347,27 @@ exports[`oda-enrollment-android-applink should render view when "No, I don’t h
                                   class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways download-ov-link emotion-23"
                                   href="https://play.google.com/store/apps/details?id=com.okta.android.auth"
                                   id="download-ov"
+                                  target="_blank"
```

Comment:
Was this intended?

Reply:
its because of this translation key: https://github.com/okta/okta-signin-widget/blob/5a886ec9d115119e2e012cae61b983307dc38f27/packages/%40okta/i18n/src/properties/login.properties#L1264

Reply:
This wasn't intentional, but since I added the `target` attribute to the allow list in dompurify, this anchor tag is now correctly getting the `target` attr as it was intended to.  I looked at this view aswell and it looks and works fine.

Reply:
Ah perfect! So this is technically a fix :D great catch!